spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/rxjava/RxJavaProcessor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.annotation.rxjava;
+
+import rx.Observable;
+
+/**
+ * Marker interface that RxJava module uses to provide the implementation bean.
+ *
+ * @author Mark Pollack
+ * @author Ilayaperumal Gopinathan
+ */
+public interface RxJavaProcessor<I, O> {
+
+	Observable<O> process(Observable<I> input);
```

Comment:
+1 let's try to get this in as is, though, since it brings Rx support at par with XD, and we can refactor later.